### PR TITLE
feat: setup v2 sitemap redirects to v3

### PR DESCRIPTION
**Pull request by paulbalaji**
resolves https://github.com/hyperlane-xyz/issues/issues/992

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -44,8 +44,315 @@ const config = {
...
+          },
+          {
+            to: '/docs/reference/applications/warp-routes',
+            from: [
+              '/docs/apis-and-sdks/warp-api',
+              '/docs/build-with-hyperlane/examples',
+              '/docs/build-with-hyperlane/quickstarts/hyperlane-quickstart',
+              '/docs/apis-and-sdks/building-applications',
+              '/docs/apis-and-sdks/building-applications/example-usage',
+              '/docs/apis-and-sdks/building-applications/nodejs-sdk',
```
Comment (paulbalaji) [inline docusaurus.config.js:83]: Could link instead to the typescript-sdk in monorepo, but then you lose consistency with other */nodejs-sdk* links

**Comment by paulbalaji** [comment]
I've noticed in general that there are several pages that are nothing but a title - should we remove those and redirect to something else or leave as-is and consider it placeholder for when we do get round to filling them out?

**Review by avious00 — APPROVED** [review]

**Comment by avious00** [comment]
> I've noticed in general that there are several pages that are nothing but a title e.g. anything at `/docs/reference/typescript-sdk`. Should we remove those and redirect to something else or leave as-is and consider it placeholder for when we do get round to filling them out?

the two pockets left are the typescript-sdk section which I expect us to come back to in two sprints, and the ecosystem section which @il-figlio @Skunkchain or I should fill out as stubs in the next few weeks. wouldn't take out for now.

let me know if i missed anything

**Comment by paulbalaji** [comment]
SDK callout
image

Made this text partial and populated the sdk pages as well so that it's not completely blank

File: docs/partials/ts-sdk-placeholder.mdx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+For developers looking to further customize the Warp UI for their own projects or build their own interchain applications, we encourage you to explore the [Typescript SDK](https://github.com/hyperlane-xyz/hyperlane-monorepo/tree/main/typescript/sdk).
+
+While we acknowledge that getting to grips with the SDK is currently a complex task, it can be a great starting point for those who are up for the challenge. We also welcome contributions and feedback from our developer community.
```
Comment (jmrossy) [inline docs/partials/ts-sdk-placeholder.mdx:3]: I don't think the self-dunk is beneficial. Maybe just "The SDK contains a growing set of a utilities and abstractions for interacting with Hyperlane across different protocols (EVM, Cosmos, Solana)."

**Review by jmrossy — APPROVED** [review]

**Pull request merged**